packages/mui-base/src/Select/Select.types.ts

```diff
@@ -2,8 +2,9 @@ import * as React from 'react';
 import { Simplify } from '@mui/types';
 import { SelectValue, UseSelectButtonSlotProps, UseSelectListboxSlotProps } from '../useSelect';
 import { SelectOption } from '../useOption';
-import Popper, { PopperProps } from '../Popper';
+import { PopperProps } from '../Popper';
 import { PolymorphicProps, SlotComponentProps, WithOptionalOwnerState } from '../utils';
+import Popper from '../Popper/Popper';
```

Comment:
```suggestion
import Popper, { PopperProps } from '../Popper';
import { PolymorphicProps, SlotComponentProps, WithOptionalOwnerState } from '../utils';
```

No need for any changes here.